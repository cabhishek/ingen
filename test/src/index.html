<!--
{
  "layout": "default",
  "title": "Home"
}
-->

<p>This is the home page.</p>

<ul>
  {{#eachArticle}}
    <li><a href="{{ permalink }}">{{title}}</a></li>
  {{/eachArticle}}
</ul>
