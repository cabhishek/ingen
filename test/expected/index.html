<!DOCTYPE html>
<html>
  <head>
    <title>InGen</title>
  </head>
  <body>
    <header>InGen</header>

    <main>
<p>This is the home page.</p>

<ul>

    <li><a href="/articles/article-two/index.html">Article Two</a></li>

    <li><a href="/articles/article-one/index.html">Article One</a></li>

</ul>
</main>
    <footer>&copy; 2013</footer>

  </body>
</html>
